theme

```diff
@@ -36,6 +36,7 @@
   "home-assistant-community-themes/teal",
   "home-assistant-community-themes/vaporwave-pink",
   "houtknots/UglyChristmas-Theme",
+  "JOHLC/transparentblue",
```

Comment:
Adding transparentblue repo to theme file

Reply:
Why are you making a review?